### DISCUSSION TITLE
Nextflow exiting without error, ".command.run" can't open file but file exists

### DISCUSSION THREAD

**Discussion by zillurbmb51** [nextflow-io/nextflow#2323] [Q&A]
Hi,

I was trying to use nextflow (v20.10.0) for multiple plink commands. Nextflow exiting after few seconds, exit status 1.
When I try to check the working directory ".command.run" it can't open file, but the input file exists.

I can see it is running giving me the plink logs, but just do not understand why it is exiting.
The same script worked multiple times, in some other directories without error. 
Just do not understand what is the issue here. Same type of input, same script. The most frustrating, it is not showing what is wrong here.
When I delete everything, and start again the error same but this time it ask to check `.command.sh` script. I checked that script and it runs without any error. So, I run again the whole nextflow script, it exit again with exit status 1 but this time does not ask to show .`command.sh` . Strange! Any way to diagnose what is happening here? 
 Any help?
[nextflow_exiting.txt](https://github.com/nextflow-io/nextflow/files/7173711/nextflow_exiting.txt)
![nextflow-exiting-status-1](https://user-images.githubusercontent.com/12833907/133520213-5b35d47a-a050-4340-a425-78f0e8fbfd30.png)
![file-exists-but-nextflow-exit-without-error](https://user-images.githubusercontent.com/12833907/133520221-b9a42df5-e7af-4646-a9f6-5f7476f3fc75.png)
[pre_phasing_to_phasing.nf.txt](https://github.com/nextflow-io/nextflow/files/7173715/pre_phasing_to_phasing.nf.txt)

**Comment by sgavril** [comment]
hey did you ever figure this out? Having the same error
